apps/user/tests/test_apis.py

```diff
@@ -216,3 +220,49 @@ def test_user_preference_feature_avialable_for_all(self):
         self.authenticate(user_fhx)
         response = self.client.get('/api/v1/users/me/preferences/')
         self.assertEqual(len(response.data['accessible_features']), 1)
+
+    def test_password_change(self):
+        user = User.objects.create_user(
+            username='ram@dave.com',
+            first_name='Ram',
+            last_name='Dave',
+            password=self.user_password,
+            email='ram@dave.com',
+        )
+        new_pass = "nepal!@#RRFASF"
+        data = {
+            "old_password": self.user_password,
+            "new_password": new_pass
+        }
+        url = '/api/v1/password-change/'
+        self.authenticate(user)
+        response = self.client.post(url, data)
+        self.assert_201(response)
+
+        user.refresh_from_db()
+        assert user.check_password(data['new_password'])
+
+        # now try with posting  diferent `new_password` that doesnot follow django password validation
+        data = {
+            "old_password": new_pass,  # since password is already changed in the database level
+            "new_password": "nepa"
+        }
+        self.authenticate(user)
+        response = self.client.post(url, data)
+        print(response.content)
```

Comment:
Let's make sure to https://github.com/the-deep/server/blob/develop/.github/pull_request_template.md#make-sure-there-are-no